README.md

```diff
@@ -31,10 +31,10 @@ This project uses the [pyrex](https://github.com/garmin/pyrex) tool to transpare
     git submodule update --remote --checkout
     ```
 
-2. #### Install docker
+1. #### Install docker
```

Comment:
I would normally agree with you that we should use auto-numbering; but there is a significant demographic of developers who will read this README in a terminal. So I would prefer to keep the numbering explicit in the markdown, even though it is a little more work.

Reply:
Done